Mergin/utils.py

```diff
@@ -186,3 +187,17 @@ def send_logs(username, logfile):
         return log_file_name, None
     except (HTTPError, URLError) as e:
         return None, str(e)
+
+
+def validate_mergin_url(url):
+    url_ping = urllib.parse.urljoin(url, urllib.parse.quote("/ping"))
```

Comment:
I would suggest to use mergin client's `is_server_compatible()` with a temporary MerginClient object (auth not needed) rather than doing the query directly.

And we don't need to deal with "online" status - if the server is in maintenance mode, that's okay (it's only a temporary thing).